packages/shared/components/Notification/Notification.tsx

```diff
@@ -92,11 +90,10 @@ export function Notification(props: NotificationProps) {
         }
       }}
       onClick={toggleIsExpanded}
+      {...props}
```

Comment:
This emits a warning because it passes design system props as well as `NotificationProps` to a `<div>`:

```
Warning: Unknown event handler property `onRemove`. It will be ignored.
    in div (created by Notification__Container)
```

The solution would be to destructure props into individual NotificationProps and then capture the rest of them into a separate var that can be spread here.

I don't have time to fix this right now but I can try to do it next week.

Reply:
https://github.com/gravitational/webapps/pull/1387